README.md

```diff
@@ -1,19 +1,18 @@
+[![Snap Status](https://build.snapcraft.io/badge/reicast/reicast-emulator.svg)](https://build.snapcraft.io/user/reicast/reicast-emulator)
+
 reicast
 ===========
-reicast is a multi-platform Sega Dreamcast emulator.
+**reicast** is a multi-platform Sega Dreamcast emulator.
 
 This is a developer-oriented resource, if you just want bins head over to http://reicast.com/
 
 For development discussion, join [#reicast in freenode](https://webchat.freenode.net/?channels=reicast)
  or stop by the [reicast Discord server](http://discord.gg/Hc6CF72)
 
-Caution
```

Comment:
Why are you removing this?

Reply:
There's no need to add such a caution these days. We are making things more documented and stable. No need to scare off ppl